Makefile

```diff
@@ -320,7 +320,9 @@ analyze: $(JS)
 # Update dependency repos
 .PHONY: update
 update: .gitmodules
-	@git submodule update --init --recursive
+	@if ! [ -f ${JERRY_BASE}/CMakeLists.txt ]; then \
+		git submodule update --init --recursive; \
+	fi
 
 ${JERRY_BASE}/CMakeLists.txt: update
```

Comment:
maybe you can test is the other line

${JERRY_BASE}/CMakeList:
     [ -f $@ ] || ${MAKE} update

I should have done it in earlier patch